### PR TITLE
Linux101: Add a landing page

### PR DESCRIPTION
**Pull request by core-man**
Related to #67.

### CODE REVIEW

File: source/computer/linux101.rst [hunk 1]
```diff
@@ -1,11 +1,61 @@
...
+      - http://billie66.github.io/TLCL/：托管在 GithHub 上的中文版本
+
+    - 作者： William Shotts
+    - 内容：系统地介绍 Linux 基础
+
+6.  UNIX Tutorial for Beginners
+
+    - 主页：http://www.ee.surrey.ac.uk/Teaching/Unix/
+    - 作者：未知
+    - 内容：UNIX 入门基础
```
Comment (seisman) [inline source/computer/linux101.rst:61]: I think we can remove these tutorials.

File: source/computer/linux101.rst [hunk 2]
```diff
@@ -1,11 +1,61 @@
...
+    - 主页：https://101.lug.ustc.edu.cn/
+    - 作者：中国科学技术大学 Linux 用户协会
+    - 内容：系统地介绍 Linux 基础
+
+3.  Bash 脚本教程
+
+    - 主页：目前存在两个网站，尚不确定哪个是未来主要维护版本
+
+      - https://www.bookstack.cn/books/bash-tutorial：书栈网版本，内容似乎更全
+      - https://wangdoc.com/bash/：网道网站，正在建设中
```
Comment (seisman) [inline source/computer/linux101.rst:35]: This is the repository for the wangdoc site. I think this one is well maintained.

Comment (core-man) [reply source/computer/linux101.rst:35]: ?? you mean only keep the second one? but it is still under development, and some content of the first doc have not been migrated to wangdoc.

File: source/computer/linux101.rst (hunk 1 above)
Comment (core-man) [reply source/computer/linux101.rst]: done in 896a8c4.

File: source/computer/linux101.rst (hunk 2 above)
Comment (core-man) [reply source/computer/linux101.rst]: Updated in 9a3ef2e.

**Review by seisman — APPROVED** [review]

File: source/computer/linux101.rst [hunk 3]
```diff
@@ -1,11 +1,38 @@
...
    :maxdepth: 2
+   :hidden:
 
    command-line 
    file-operations
    environment-variable
+
+:本节贡献者: |姚家园|\（作者）、
+             |田冬冬|\（审稿）
+:最近更新日期: 2021-03-02
```
Comment (core-man) [inline source/computer/linux101.rst:14]:
```suggestion
:最近更新日期: 2021-03-22
```

File: source/computer/linux101.rst [hunk 4]
```diff
@@ -1,11 +1,38 @@
...
+   :hidden:
 
    command-line 
    file-operations
    environment-variable
+
+:本节贡献者: |姚家园|\（作者）、
+             |田冬冬|\（审稿）
+:最近更新日期: 2021-03-22
+:预计花费时间: 10 分钟
```
Comment (core-man) [inline source/computer/linux101.rst:15]:
```suggestion
:预计花费时间: 2 分钟
```

**Pull request merged**